### ISSUE TITLE
Using generate:entity:content breaks site after installing the module.

### ISSUE THREAD

**Issue by gonssal** [hechoendrupal/drupal-console#4065]
### Problem/Motivation
generate:entity:content breaks site after the module installation.

### How to reproduce
Generate a module and then use generate:entity:content on it. Install the module afterwards.

After installation, I get the following error:

```
[ERROR] SQLSTATE[42S02]: Base table or view not found: 1146 Table 'drupal8.animal' doesn't exist: SELECT base_table.vid
         AS vid, base_table.id AS id                                                                                    
         FROM                                                                                                           
         {animal} base_table                                                                                            
         GROUP BY base_table.vid, base_table.id                                                                         
         LIMIT 1 OFFSET 0; Array                                                                                        
         (                                                                                                              
         )
```
The created entity name is "animal" of course.

The error also happens when trying to uninstall the module, even with Drupal Console or Drush, so basically the site is broken.

Also I tried using `drupal update:entities` as stated in Drupal's documentation, to no avail.

- Drupal version: 8.7.2
- Console version: 1.8.0

**Comment by gonssal** [comment]
Closing because after some digging it seems more like a core bug to me and I've given up on D8 for the project.

**Comment by stephenplatz** [comment]
This exact error is happening for me.

- Drupal version 8.7.3
- Console version 1.9.1

**Comment by gonssal** [comment]
So I stumbled upon this again in another project.

I haven't digged into it, but to me it seems that the issue is [this change in core](https://www.drupal.org/node/3034742). This makes `drush entup` not work for <=8.7 and I bet drupal-console has the same problem. The latest versions of Drush already warn about this and don't run the `entup` command.

With the [devel_entity_updates](https://www.drupal.org/project/devel_entity_updates) project, you can make the Drush command work again, but read the warnings, because the feature was removed for a reason.

For Drupal console, I think the only way is to wait for the devs to either implement what's in devel_entity_updates or just remove the `drupal update:entities` command and add a warning to the `generate:entity:content` saying that you have to implement the schema changes yourself.

Hope it helps someone.

**Comment by enzolutions** [comment]
Fixed with PR https://github.com/hechoendrupal/drupal-console/pull/4155 will be included in release 1.9.4
